shared/tracker/header.render.desktop.js

```diff
@@ -8,32 +8,22 @@ import type {HeaderProps} from './header.render'
 
 export default class HeaderRender extends Component {
   props: HeaderProps;
-  state: {showCloseWarning: boolean};
-
-  constructor (props: HeaderProps) {
-    super(props)
-    this.state = {showCloseWarning: false}
-  }
 
   render () {
-    let headerStyle = (this.props.currentlyFollowing && !this.props.changed) ? styles.headerSuccess : styles.headerNormal
+    // $FlowFixMe // how we split these props needs to change
+    let headerStyle = this.props.backgroundStyle
```

Comment:
i only hoisted up the background changing as thats now shared between the header and the bio